articles/support/pre-deployment/index.html

```diff
@@ -0,0 +1,39 @@
+---
+url: /support/pre-deployment-tests
+classes: topic-page
+title: Pre-Deployment Tests
+---
+
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>Pre-Deployment Tests</h1>
+  <p>
+    Before you go live, run Auth0's pre-deployment test suite to ensure that your tenants are ready for use in a production environment.
+  </p>
+</div>
+
+<ul class="topic-links">
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/support/pre-deployment/how-to-run-test">How to Run the Pre-Deployment Tests</a>
+    <p>
+      Run the pre-deployment tests against one or more clients to see if they're production-ready or not.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i>Pre-Deployment Test Results
+    <p>
+      View your results to see any outstanding required and recommended tasks, as well changes you can make to ensure that you comply with best practices.
+    </p>
+    <ul>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/support/pre-deployment/tests/required">Required Fixes</a>
+      </li>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/support/pre-deployment/tests/recommended">Recommended Fixes</a>
+      </li>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/support/pre-deployment/tests/best-practice">Fixes to Comply with Best Practices</a>
+      </li>
+    </ul>
+  </li>
+</ul>
```

Comment:
Do you think we should also move this doc here? https://auth0.com/docs/prelaunch-tips I think it makes more sense that the `Getting Started` section